src/events.js

```diff
@@ -73,11 +74,42 @@ export const getKinesisData = event => {
   return { arn, messageIds };
 };
 
+export const getDynamodbData = event => {
+  const arn = event.Records[0].eventSourceARN;
+  const approxEventCreationTime =
+    event.Records[0].dynamodb.ApproximateCreationDateTime;
+  const messageIds = (event.Records || [])
+    .map(record => {
+      if (
+        record.eventName === 'MODIFY' &&
+        record.dynamodb &&
+        record.dynamodb.Keys
+      ) {
+        return md5Hash(record.dynamodb.Keys);
+      } else if (
+        record.eventName === 'REMOVE' &&
+        record.dynamodb &&
+        record.dynamodb.Keys
+      ) {
+        return md5Hash(record.dynamodb.Keys);
+      } else if (
+        record.eventName === 'INSERT' &&
+        record.dynamodb &&
+        record.dynamodb.NewImage
+      ) {
+        return md5Hash(record.dynamodb.NewImage);
+      }
+    })
+    .filter(x => !!x);
```

Comment:
Why we need this ?

Reply:
if one of the records is not INSERT, MODIFY, REMOVE then it will be undefined - so we want to remove it from the list